cluster/node-pools/worker-default/userdata.clc.yaml

```diff
@@ -214,6 +214,7 @@ systemd:
       --node-labels=${REPLACEMENT_STRATEGY_LABEL} \
 {{- end }}
       --node-labels={{ .Values.node_labels }} \
+      --node-labels=beta.kubernetes.io/distro=coreos \
```

Comment:
Just a note: Depending on when we merge this we may want to avoid rolling the old nodes at the same time. Could be solved by using `NotIn` on the daemonset.

Reply:
Done